pkg/controllers/disruption/controller.go

```diff
@@ -283,7 +282,7 @@ func (c *Controller) logAbnormalRuns(ctx context.Context) {
 func (c *Controller) logInvalidBudgets(ctx context.Context) {
 	nodePoolList := &v1beta1.NodePoolList{}
 	if err := c.kubeClient.List(ctx, nodePoolList); err != nil {
-		logging.FromContext(ctx).Errorf("listing nodepools, %s", err)
+		log.FromContext(ctx).Error(err, "listing nodepools, %s")
```

Comment:
We should take a look at all of these error loggings that we are doing now. They have formatting directives at the end of them, which will no longer apply if the error is getting explicitly passed-through as an argument